proto/user_stats.proto

```
@@ -1,11 +1,19 @@
 syntax = "proto3";
 
+import "google/protobuf/timestamp.proto";
+
 package user_stats;
```

Buf (1.57.2)

Files with package "user_stats" must be within a directory "user_stats" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)